src/backend/renderer/gles2/mod.rs

```diff
@@ -487,16 +490,10 @@ impl Gles2Renderer {
             if gl_version < version::GLES_3_0 && !exts.iter().any(|ext| ext == "GL_EXT_unpack_subimage") {
                 return Err(Gles2Error::GLExtensionNotSupported(&["GL_EXT_unpack_subimage"]));
             }
-            // required for instanced damage rendering
-            if gl_version < version::GLES_3_0
-                && !(exts.iter().any(|ext| ext == "GL_EXT_instanced_arrays")
-                    && exts.iter().any(|ext| ext == "GL_EXT_draw_instanced"))
-            {
-                return Err(Gles2Error::GLExtensionNotSupported(&[
-                    "GL_EXT_instanced_arrays",
-                    "GL_EXT_draw_instanced",
-                ]));
-            }
+            // Check if GPU supports instanced rendering.
+            let supports_instancing = gl_version >= version::GLES_3_0
```

Comment:
Could it be useful to have an environment variable for all us with high end cards to test this?

Reply:
See https://github.com/Smithay/smithay/pull/495#discussion_r805519150.

I wrote this patch myself on a computer that supports instanced rendering. I just don't see why it would be any easier to set an environment variable compared to just changing the code when you're compiling from source anyway.